plaso/formatters/gdrive_synclog.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""Google Drive Sync log event formatter."""
+
+from __future__ import unicode_literals
+
+from plaso.formatters import interface
+from plaso.formatters import manager
+
+
+class GDriveSyncLogFormatter(interface.ConditionalEventFormatter):
```

Comment:
GDrive => GoogleDrive

Reply:
I've expanded this in all/most places -- I left the filenames and data-type declarations as-is, since that felt consistent with the existing Google Drive parsers. Shout if that's something you want me to revise further or you see anywhere else I missed it.